graphql/host_resolver.go

```diff
@@ -0,0 +1,81 @@
+package graphql
+
+// This file will be automatically regenerated based on the schema, any resolver implementations
+// will be copied through when generating and any unknown code will be moved to the end.
+
+import (
+	"context"
+	"fmt"
+	"time"
+
+	"github.com/evergreen-ci/evergreen/model/host"
+	restModel "github.com/evergreen-ci/evergreen/rest/model"
+	"github.com/evergreen-ci/utility"
+	"github.com/mongodb/grip"
+	"github.com/mongodb/grip/message"
+	werrors "github.com/pkg/errors"
```

Comment:
Does this need to be werrors if we aren't usin ga different errors package?

Reply:
I think it's an issue with `gqlgen`. 😭 If I don't give it a specific name, it'll auto-import the `"errors"` package instead of `"github.com/pkg/errors"` package every time I run `make gqlgen`. 

There's actually an open issue about this in `99designs/gqlgen` repo and I left a comment in `resolver.go` linking to it. (I don't think I can leave a comment in the more specific resolver files because they get overwritten every time I generate)

Reply:
Another option is we could switch to using fmt since we just use .Error anyway (like instead of `werrors.Wrapf(err, "Problem getting last known project for '%s'", alias.ChildProject).Error())` we could do `fmt.Sprintf("Problem getting last known project for '%s': %s", alias.ChildProject, err.Error())`
But whatever works!